config/initializers/devise.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
```

Comment:
In general, if you leave the config options commented out, you don't need the comments explaining it. 

I like explaining my configuration choices, in the commit history, but it's not exactly necessary.